tests/integration/test_end_to_end_pipeline.py

```
@@ -0,0 +1,132 @@
+    pkg.__path__ = [str(Path(__file__).resolve().parents[2] / "src" / "trading_rl_agent")]
+    sys.modules["trading_rl_agent"] = pkg
+
+if "nltk.sentiment.vader" not in sys.modules:
+    dummy = types.ModuleType("nltk.sentiment.vader")
+    class DummySIA:
+        def polarity_scores(self, text):
+            return {"compound": 0.0}
+    dummy.SentimentIntensityAnalyzer = DummySIA
+    sys.modules["nltk.sentiment.vader"] = dummy
+
+import pandas as pd
```

Ruff (0.11.9)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)